lnd.go

```diff
@@ -634,11 +639,13 @@ func genCertPair(certFile, keyFile string) error {
 		}
 	}
 
-	// Add extra IPs to the slice.
-	for _, ip := range cfg.TLSExtraIPs {
-		ipAddr := net.ParseIP(ip)
-		if ipAddr != nil {
-			addIP(ipAddr)
+	if cfg != nil {
```

Comment:
Instead of checking the global `cfg` var here, you could pass `TLSExtraIPs` and `TLSExtraDomains` as arguments to the method instead.